pyspectral/rayleigh.py

```diff
@@ -237,7 +239,20 @@ def _do_interp(minterp, sunzsec, azidiff, satzsec):
         res = minterp(interp_points2)
         return res.reshape(sunzsec.shape)
 
-    def get_reflectance(self, sun_zenith, sat_zenith, azidiff, bandname, redband=None):
+    @staticmethod
+    def _do_reduce_effect(zenith, rayref, thresh_zen, maxzen, strength):
```

Comment:
Could we have a more specific name, like eg: `_reduce_rayleigh_correction_at_high_zenith_angles`?